call_test.go

```diff
@@ -285,7 +285,7 @@ func TestInvokeCancelClosedNonFailFast(t *testing.T) {
 	req := "hello"
 	ctx, cancel := context.WithCancel(context.Background())
 	cancel()
-	if err := Invoke(ctx, "/foo/bar", &req, &reply, cc, FailFast(false)); err == nil {
+	if err := Invoke(ctx, "/foo/bar", &req, &reply, cc); err == nil {
```

Comment:
this test is called `TestInvokeCancelClosedNonFailFast` - did you mean to make this change?

Reply:
Good catch. Reverted.